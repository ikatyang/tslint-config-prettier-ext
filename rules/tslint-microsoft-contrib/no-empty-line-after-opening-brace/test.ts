if (true) {

  console.log('something');
}
