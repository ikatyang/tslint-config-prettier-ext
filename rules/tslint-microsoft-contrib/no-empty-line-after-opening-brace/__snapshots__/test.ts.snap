// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should affect error message after formatting 1`] = `
"
<<<<<< before
ERROR: (no-empty-line-after-opening-brace) /rules/tslint-microsoft-contrib/no-empty-line-after-opening-brace/test.ts[2, 1]: Opening brace cannot be followed by empty line

======
no error

>>>>>> after
"
`;

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
if (true) {

  console.log('something');
}

======
if (true) {
  console.log(\\"something\\");
}

>>>>>> after
"
`;
